script/sheet/weapon.js

```diff
@@ -4,6 +4,47 @@ export class ForbiddenLandsWeaponSheet extends ForbiddenLandsItemSheet {
   static get defaultOptions() {
     return mergeObject(super.defaultOptions, {
       template: "systems/forbidden-lands/model/weapon.html",
+      tabs: [
+        {
+          navSelector: ".sheet-tabs",
+          contentSelector: ".sheet-body",
+          initial: "attributes",
```

Comment:
Should probably be changed to `main`, right? Although this doesn't seem to actually matter.